samples/azure-quantum/resource-estimation/estimation-factoring.ipynb

```diff
@@ -177,21 +177,23 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "experiments = [\n",
-    "    (\"Gate-based µs, 10⁻³\", {\"qubitParams\": {\"name\": \"qubit_gate_us_e3\"}, \"errorBudget\": 0.333}),\n",
-    "    (\"Gate-based µs, 10⁻⁴\", {\"qubitParams\": {\"name\": \"qubit_gate_us_e4\"}, \"errorBudget\": 0.333}),\n",
-    "    (\"Gate-based ns, 10⁻³\", {\"qubitParams\": {\"name\": \"qubit_gate_ns_e3\"}, \"errorBudget\": 0.333}),\n",
-    "    (\"Gate-based ns, 10⁻⁴\", {\"qubitParams\": {\"name\": \"qubit_gate_ns_e4\"}, \"errorBudget\": 0.333}),\n",
-    "    (\"Majorana ns, 10⁻⁴\", {\"qecScheme\": {\"name\": \"floquet_code\"}, \"qubitParams\": {\"name\": \"qubit_maj_ns_e4\"}, \"errorBudget\": 0.333}),\n",
-    "    (\"Majorana ns, 10⁻⁶\", {\"qecScheme\": {\"name\": \"floquet_code\"}, \"qubitParams\": {\"name\": \"qubit_maj_ns_e6\", \"tGateErrorRate\": 0.01}, \"errorBudget\": 0.333})\n",
+    "labels = [\"Gate-based µs, 10⁻³\", \"Gate-based µs, 10⁻⁴\", \"Gate-based ns, 10⁻³\", \"Gate-based ns, 10⁻⁴\", \"Majorana ns, 10⁻⁴\", \"Majorana ns, 10⁻⁶\"]\n",
+    "\n",
+    "items = [\n",
+    "    {\"qubitParams\": {\"name\": \"qubit_gate_us_e3\"}, \"errorBudget\": 0.333},\n",
```

Comment:
Is it worth making a constant for error budget?

Reply:
Items is the terminology to refer to an entity in a batching job, so I prefer to keep that identifier here.

Reply:
I am updating this and the other notebooks in a separate PR.